setup.cfg

```diff
@@ -14,7 +14,6 @@ exclude_lines =
     return NotImplemented
 
 [tool:pytest]
-addopts = -n auto --vcr-record-mode=none
```

Comment:
Don't forget to update tox with `-n auto`

Reply:
It was added by mistake. `git stash` is guilty!